### PR TITLE
switch policy-controller to latest v0.2.0

### PR DESCRIPTION
**Pull request by hectorj2f**
Signed-off-by: hectorj2f <hectorf@vmware.com>

 **Description of the change**

Bump policy-controller to pull the latest version from its new repository v0.2.0

 **Existing or Associated Issue(s)**

 **Additional Information**

 

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [x] Variables are documented in the `values.yaml` and added to the README.md. THe [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content
- [x] JSON Schema generated
- [x] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command

### CODE REVIEW

**Review by cpanato — DISMISSED** [review]

File: charts/policy-controller/kubeconfig [hunk 1]
```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```
Comment (cpanato) [inline charts/policy-controller/kubeconfig:1]: do we need this?

File: charts/policy-controller/README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ The following table lists the configurable parameters of the policy-controller c
 | policywebhook.extraArgs | object | `{}` |  |
 | policywebhook.image.pullPolicy | string | `"IfNotPresent"` |  |
 | policywebhook.image.repository | string | `"gcr.io/projectsigstore/policy-webhook"` |  |
```
Comment (cpanato) [inline charts/policy-controller/README.md:23]: should we update this as well to `ghcr.io/sigstore/policy-controller`?

**Review by cpanato — COMMENTED** [review]
thanks for doing this, a few comments

File: charts/policy-controller/kubeconfig (hunk 1 above)
Comment (hectorj2f) [reply charts/policy-controller/kubeconfig]: 🤦🏻 , it is gone now.

File: charts/policy-controller/Chart.yaml [hunk 3]
```diff
@@ -19,6 +19,6 @@ annotations:
   artifacthub.io/license: Apache-2.0
   artifacthub.io/images: |
     - name: policy-controller
-      image: gcr.io/projectsigstore/policy-controller:v1.9.0@sha256:87dffbad3e3d7a7ce9cc4184607240df8f39943db25772e0e4b2800deb7e5134
+      image: ghcr.io/sigstore/policy-controller/policy-controller:v0.2.0@sha256:63cd54b877bcd8b3d6d6b7110955e3f773bd8ad5b16d1b6133d4009c26c6da18
```
Comment (hectorj2f) [inline charts/policy-controller/Chart.yaml:22]: i changed the repository. It looked weird the image digest to the same repo but pointing to a lower version.

**Review by cpanato — APPROVED** [review]
thank you!

**Pull request merged**